src/AppBundle/Resources/javascript/Client/NotificationClient.ts

```diff
@@ -7,6 +7,12 @@ export class NotificationClient {
    * Request status form the API.
    */
   public send(callback: (result: string) => void, errorCallback: (error: unknown) => void) {
+    jQuery.ajaxSetup(
+      {
+        // Wait max 10 seconds.
```

Comment:
My question in the future would be: why wait max 10 seconds? At this moment I know this is to prevent crashes in Safari. My advice is to add extra feedback in this comment or open a bug ticket where this specific problem is described.